var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -22,7 +22,9 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+
```

Comment:
Seems nice to have it -- there's a long copyright message above, and the import statements are rather short in comparison, and are not related to the copyright statement in any way.